target/linux/ath79/dts/qca955x_ubnt_xc.dtsi

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: GPL-2.0
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca955x.dtsi"
+
+/ {
+	compatible = "ubnt,xc", "qca,qca9557";
+	model = "Ubiquiti Networks XC board";
+};
```

Comment:
I'd drop these two proporties entirely. model makes no sense except for a device, and while I would have argued about the compatible, it makes not much sense either with the file being qca955x and the value being qca,qca9557 ...
We are not using it anyway ...

Reply:
I got it from the WA boards dtsi:
```
/ {
	compatible = "ubnt,wa", "qca,ar9342";
	model = "Ubiquiti Networks WA board";

```
but, I agree, there is no such device.